astropy/modeling/tests/test_math_func.py

```diff
@@ -0,0 +1,18 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import numpy as np
+from numpy.testing import assert_allclose#, assert_quantity_allclose
+
+from .. import math_functions
+
+x = np.linspace(-20, 360, 100)
+y = np.linspace(-20, 360, 100)
+
+def test_math():
```

Comment:
Perhaps for more persuasiveness, use ufunc= getattr(np, name) and test against that instead of model.func?